content/docs/latest/container-runtimes/incus.md

```diff
@@ -0,0 +1,239 @@
+PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
+UBUNTU_CODENAME=jammy
+```
+
+# Using Incus and ZFS
+
+[ZFS][zfs] is provided as a Systemd sysext Flatcar extension as well, it fits perfectly with Incus as a storage driver. This configuration can be used to deactivate Docker and Containerd and enable Incus runtime using ZFS storage.
+
+```yaml
+---
+# config.yaml
+# butane < config.yaml > config.json
+variant: flatcar
+version: 1.1.0
+storage:
+  files:
+    - path: /etc/flatcar/enabled-sysext.conf
+      contents:
+        inline: |
+          incus
+          zfs
+    - path: /etc/incus/pre-seed.yaml
+      contents:
+        inline: |
+          config: {}
+          networks:
+          - config:
+              ipv4.address: auto
+              ipv6.address: auto
+            description: ""
+            name: incusbr0
+            type: ""
+            project: default
+          storage_pools:
+          - config:
+              size: 5GiB
+            description: "Incus ZFS pool"
+            name: default
+            driver: zfs
+          profiles:
+          - config: {}
+            description: ""
+            devices:
+              eth0:
+                name: eth0
+                network: incusbr0
+                type: nic
+              root:
+                path: /
+                pool: default
+                type: disk
+            name: default
+          projects: []
+          cluster: null
+  links:
+    - path: /etc/extensions/docker-flatcar.raw
+      target: /dev/null
+      overwrite: true
+    - path: /etc/extensions/containerd-flatcar.raw
+      target: /dev/null
+      overwrite: true
+systemd:
+  units:
+    - name: incus-admin-init.service
+      enabled: true
+      contents: |
+        [Unit]
+        Description=incus admin init
+        After=systemd-sysext.service
+        Requires=systemd-sysext.service
+        [Service]
+        StandardInput=file:/etc/incus/pre-seed.yaml
+        Type=oneshot
+        RemainAfterExit=yes
+        ExecStart=/usr/bin/incus admin init --preseed
+        [Install]
+        WantedBy=multi-user.target
+```
+
+Incus will now use ZFS pool to store images and containers:
+
+```bash
+core@localhost ~ $ incus storage list
++---------+--------+----------------+---------+---------+
+|  NAME   | DRIVER |  DESCRIPTION   | USED BY |  STATE  |
++---------+--------+----------------+---------+---------+
+| default | zfs    | Incus ZFS pool | 1       | CREATED |
++---------+--------+----------------+---------+---------+
+core@localhost ~ $ zpool list
+NAME      SIZE  ALLOC   FREE  CKPOINT  EXPANDSZ   FRAG    CAP  DEDUP    HEALTH  ALTROOT
+default  4.50G   592K  4.50G        -         -     1%     0%  1.00x    ONLINE  -
+```
+
+
+[incus]: https://linuxcontainers.org/incus/
+[documentation]: https://linuxcontainers.org/incus/docs/main/
+[official-sysext]: https://www.flatcar.org/docs/latest/provisioning/sysext/#flatcar-release-extensions-official
```

Comment:
Do we have a rule about links to other pages in the flatcar docs? Whether these ought to be absolute links like this one here, or maybe relative ones. Relative links would point to an updated page within stage sites, but I'm not sure if the final website preserves the directory structure for the relative links to be valid in github repo and in the website.